### DISCUSSION TITLE
Editing Pokemon

### DISCUSSION THREAD

**Discussion by rockgod407** [jynnie/soullocke#16] [Ideas]
Hey! Been playing a soullocke with my brother in a different time zone. we usually just meet up at each gym, but when he goes to log his pokemon, he cant log them in route's i've already added to my team. He has to create a whole new line item. 

Can we set it so we can edit the pokemon even if we've moved on so he can edit the same line items?

**Comment by jynnie** [comment]
Hi! Yes, that's a great point, I'll get to this next week!

**Comment by rockgod407** [comment]
You're amazing!! Thank you so much!

**Comment by jynnie** [comment]
Update will work like this (will be live Monday)! Let me know if this doesn't resolve your issue, but I think this should let your brother backfill their Pokemon to the route.

![2023-02-25 16 50 08](https://user-images.githubusercontent.com/7284723/221386464-e75898c1-b2fe-4a28-98ef-4588a61fe41e.gif)
